### ISSUE TITLE
Multiclass classification

### ISSUE THREAD

**Issue by razla** [jespb/Python-M3GP#1]
Hello,

does this package support multiclass classification?

thanks!

**Comment by jespb** [comment]
Hi,

Yes, this implementation supports multiclass classification.
If you have any issue or suggestion, please let me know :)

**Comment by razla** [comment]
Thank you !

Are there any documentations for the use of this package?
Is it scikit-learn based API?

thanks again!

**Comment by jespb** [comment]
The readme.txt file contains a list of flags you may use when running this implementation in a stand-alone fashion but, currently, the only documentation about the methods themselves is within the code. ( I may update the readme.txt to also include a description about the most important methods. )

Until then, it should be fairly easy to import and use this implementation. I tried to make it scikit-learn-like in the way that you should be able to use it using the following piece of code:

```
from m3gp.M3GP import M3GP
model = M3GP()
model.fit(Train_x, Train_y, Test_x, Test_Y)
```

after this you may obtain the final model, the accuracy over the generations or predict new data using the methods in the M3GP class.

**Comment by razla** [comment]
Thank you again!

is there anyway I can change the fitness function? for research purposes..

**Comment by jespb** [comment]
Yes, you can change the fitness function in the m3gp/Individual file.

To change the fitness function, you just need to edit the "getFitness(self)" method. 
You may check the getTrainingAccuracy(self) and getTrainingPredictions(self) methods for reference

If you have any other question, feel free to ask :)

**Comment by razla** [comment]
Thanks! I'll check it out.. 
Is there any other way I can contact you? 
I'm working on a research and would love to ask you couple of questions regarding the package

**Comment by jespb** [comment]
You can send an email to jebatista@fc.ul.pt ( I'll remove this message and close this issue after that )

**Comment by razla** [comment]
thanks!
